Resources/translations/content_tree.en.xliff

```diff
@@ -11,6 +11,11 @@
         <target state="new">Collapse all</target>
         <note>key: collapse_all</note>
       </trans-unit>
+      <trans-unit id="0cba8ae8f7a758a11886b6be4e7ce7e570063f8f" resname="expand_item.limit.message">
+        <source>Cannot load subitems for given location due to performance limits. You reached max tree depth.</source>
```

Comment:
We changed this message with @DominikaK so that it is more precise. Thank you Dominika for helping me. :) 
ping @inakijv @SylvainGuittard

Reply:
Yesterday, I suggested to Iñaki:
`For performance reasons, the navigation using content tree is limited. Please use the sub-items section to continue navigation`
What do you think?